test/screens/v2/widget_instance/line_map_test.exs

```diff
@@ -4,6 +4,7 @@ defmodule Screens.V2.WidgetInstance.LineMapTest do
   alias Screens.Predictions.Prediction
   alias Screens.Schedules.Schedule
   alias Screens.Stops.Stop
+  alias Screens.Trips.Trip
```

Comment:
Doesn't look like `Trip` is used anywhere.

Reply:
I think it is (e.g. here: https://github.com/mbta/screens/pull/1070/files#diff-57482df2b19067549da87f0f7d0d88f13aa39dc250a41473aa1ade5103c2b610R110).